gobblin-utility/src/main/java/gobblin/util/hive/HiveSerDeWrapper.java

```diff
@@ -10,7 +10,7 @@
  * CONDITIONS OF ANY KIND, either express or implied.
  */
 
-package gobblin.hive;
+package gobblin.util.hive;
```

Comment:
This is a backwards incompatible change that will break anyone who depends on HiveSerDeWrapper . Why is that necessary?